### PR TITLE
Celina 5

### PR DESCRIPTION
**Pull request by CelinaRiegel**

### CODE REVIEW

File: Exercises/Celina/exercises_05.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                bag_count.append(product)
+                del counter[-1]
+                return bag_count
+            else:
+                number_bags=int(bag[0])
+                if len(counter)<=recursion_count:
+                    counter.append(1)
+                counter[recursion_count]=number_bags
+                if number_bags==1:
+                    new_start_bag=bag[1] + "s"
```
Comment (TrueEsel) [inline Exercises/Celina/exercises_05.py:122]: Smart way to resolve the problem of having "bag" and "bags"

**Review by TrueEsel — APPROVED** [review]
The code ist pretty readable and does what its supposed to. I don't know if you did that on purpose but you're deleting the exercises one and two from your folder

File: Exercises/Celina/exercises_05.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+# You can find the ACTUAL input for this exercise under `data/input_sequence.txt`. For this
+# real input you should consider not only the 5 numbers, but the 25 numbers preceding.
+# Find the first number in this list which can not be expressed as a
+# sum of two numbers out of the 25 numbers before it.
+# Please make not of your result in the PR.
+
+test =  [35, 20, 15, 25, 47, 40, 62, 55, 65, 95, 102, 117, 150, 182, 127, 219, 299, 277, 309, 576]
+
+list=[]
+text= open("data/input_sequence.txt","r")
```
Comment (ClasherKasten) [inline Exercises/Celina/exercises_05.py:22]: Use a contextmanager.

File: Exercises/Celina/exercises_05.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+# The ACTUAL input to your puzzle is given in `data/input_bags.txt`, and much larger
+# and much more deeply nested than the example above. 
+# For the actual inputs, how many bags are inside your single shiny gold bag?
+# As usual, please list the answer as part of the PR.
+
+bags_dic = {}
+
+word = " contain "
+
+bags= open("data/input_bags.txt","r")
```
Comment (ClasherKasten) [inline Exercises/Celina/exercises_05.py:82]: Use a contextmanager.

File: Exercises/Celina/exercises_05.py [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+                return bag_count
+            else:
+                number_bags=int(bag[0])
+                if len(counter)<=recursion_count:
+                    counter.append(1)
+                counter[recursion_count]=number_bags
+                if number_bags==1:
+                    new_start_bag=bag[1] + "s"
+                else:
+                    new_start_bag=bag[1]
```
Comment (ClasherKasten) [inline Exercises/Celina/exercises_05.py:124]: Use "Ternary" operator.

**Review by ClasherKasten — CHANGES_REQUESTED** [review]

**Pull request merged**